custom/icds_reports/static/js/directives/indie-map/indie-map.directive.js

```diff
@@ -191,6 +191,12 @@ function IndieMapController($scope, $compile, $location, $filter, storageService
                         .translate([element.offsetWidth / 2, element.offsetHeight / div]);
                     path = d3.geo.path().projection(projection);
                 }
+                $(function () {
+                    var svg = d3.select('#map svg');
+                    svg.selectAll(".datamaps-subunit").transition().style('fill', vm.map.fills.defaultFill);
+                    vm.addCombinedSelectorClassToMaps(document.getElementsByClassName("datamaps-subunit"));
```

Comment:
I was tempted to request a comment here explaining a bit more about this. feel free to add one in a follow up if you think it'd be helpful

Reply:
Sure